File: examples/url/list/node/listYourShortenedUrls.js
const { UrlService } = require("micro-js-client/url");

// List information on all the shortened URLs that you have created
async function listYourShortenedUrls() {
  let urlService = new UrlService(process.env.MICRO_API_TOKEN);
  let rsp = await urlService.list({});
  console.log(rsp);
}

listYourShortenedUrls();

File: examples/go/url/list/listYourShortenedUrls.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/url"
)

// List information on all the shortened URLs that you have created
func ListYourShortenedUrls() {
	urlService := url.NewUrlService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := urlService.List(&url.ListRequest{})
	fmt.Println(rsp, err)
}
